File: app/src/main/java/io/github/anvell/stackoverview/model/Question.java
package io.github.anvell.stackoverview.model;

import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;

import com.google.gson.annotations.SerializedName;

import java.net.URLDecoder;
import java.util.ArrayList;
import java.util.List;

@Entity
public class Question {

    @PrimaryKey
    @SerializedName("question_id")
    public int questionId;

    @SerializedName("tags")
    public ArrayList<String> tags = new ArrayList<>();

    @SerializedName("is_answered")
    public Boolean isAnswered;

    @SerializedName("view_count")
    public int viewCount;

    @SerializedName("answer_count")
    public int answerCount;

    @SerializedName("score")
    public int score;

    @SerializedName("creation_date")
    public int creationDate;

    @SerializedName("title")
    public String title = "";
}

File: app/src/main/java/io/github/anvell/stackoverview/model/Question.kt
package io.github.anvell.stackoverview.model

import com.google.gson.annotations.SerializedName
import java.net.URLDecoder

data class Question (
    @SerializedName("tags") val tags: List<String> = listOf(),
    @SerializedName("is_answered") val isAnswered: Boolean = false,
    @SerializedName("view_count") val viewCount: Int = 0,
    @SerializedName("answer_count") val answerCount: Int = 0,
    @SerializedName("score") val score: Int = 0,
    @SerializedName("creation_date") val creationDate: Int = 0,
    @SerializedName("question_id") val questionId: Int = 0,
    @SerializedName("title") val title: String = ""
)